ase/api/src/buffer.c

```diff
@@ -532,3 +533,40 @@ fpga_result __FPGA_API__ ase_fpgaGetIOAddress(fpga_handle handle, uint64_t wsid,
 	}
 	return result;
 }
+
+/*
+ * A single PASID is associated with the process, independent of the number
+ * of FPGA ports connected or the open/close sequence.
+ */
+static int process_pasid;
```

Comment:
May want to init to -1.  0 is a valid fd.

Reply:
process_pasid is guaranteed non-zero when valid. It isn't an fd.